spec/controllers/api/v1/education/data_controller_spec.rb

```diff
@@ -3,17 +3,31 @@
 RSpec.describe Api::V1::Education::DataController, type: :controller do
 
   describe 'get index' do
+    before(:each) { create(:state_education_data) }
     it "returns list of all data as json" do
-      create(:state_education_data)
       get :index
-      data = JSON.parse(response.body)
+      data = JSON.parse(response.body)["data"]
+
```

Comment:
I'm not sure if it was the addition of the active model serializers, but the format of the data changed... all nested under the { "data" => [ [ "id" => .... so had to modify the tests.